src/Compilers/Core/Portable/CommandLine/CommandLineParser.cs

```diff
@@ -120,6 +120,7 @@ internal static bool IsOptionName(string optionName, ReadOnlySpan<char> value)
 
             return true;
 
+#if DEBUG
```

Comment:
Likely need to also wrap the `Debug.Assert` in `#if DEBUG` otherwise it will fail compilation.

Reply:
Oh, missed the last commit, I guess that works too.